src/components/PlayerSpotlight/PlayerSpotlight.module.css

```diff
@@ -6,7 +6,7 @@
 
 .spotlightImageContainer {
     background-color:  #ff9c5a;
-    background-image: url(./text_only.webp);
+    background-image: url(../../images/text_only.webp);
```

Comment:
I moved these images into the `images` directory so all images are in one place!